src/lib/platform/exit_code.h

```diff
@@ -101,14 +102,16 @@ const char *otExitCodeToString(uint8_t aExitCode);
  * @param[in]   aExitCode   The exit code.
  *
  */
-#define VerifyOrDie(aCondition, aExitCode)                                                                   \
-    do                                                                                                       \
-    {                                                                                                        \
-        if (!(aCondition))                                                                                   \
-        {                                                                                                    \
-            otLogCritPlat("%s() at %s:%d: %s", __func__, __FILE__, __LINE__, otExitCodeToString(aExitCode)); \
-            exit(aExitCode);                                                                                 \
-        }                                                                                                    \
+#define VerifyOrDie(aCondition, aExitCode)                                                         \
+    do                                                                                             \
+    {                                                                                              \
+        if (!(aCondition))                                                                         \
+        {                                                                                          \
+            const char *start = strrchr(__FILE__, '/');                                            \
```

Comment:
```suggestion
            const char *_filename = strrchr(__FILE__, '/');                                            \
```

nit: I am worried that `start` may hide another variable in the enclosing blocks.

Reply:
I think it's fine as it's a scoped variable.